Dashboard/Views/ConnectionWindow.axaml.cs

```diff
@@ -0,0 +1,68 @@
+using Avalonia;
+using Avalonia.Controls;
+using Avalonia.Interactivity;
+using Avalonia.Markup.Xaml;
+using Avalonia.Media;
+using Dashboard.ViewModels;
+using DynamicData;
+using DynamicData.Kernel;
+using System;
+using System.ComponentModel;
+using System.IO.Ports;
+using System.Runtime.CompilerServices;
+
+namespace Dashboard;
+
+public partial class ConnectionWindow : Window
+{
+    public ConnectionWindow()
+    {
+        InitializeComponent();
+       
+    }
+
+
+    private void InitializeComponent()
+    {
+        AvaloniaXamlLoader.Load(this);
+
+    }
+
+    protected override void OnClosed(EventArgs e)
+    {
+        base.OnClosed(e);
+
+    }
+
+    private void RefreshPorts(object? sender, RoutedEventArgs? e)
```

Comment:
Adding onto Nicks comments, this function currently adds on top of what is already present in the ComboBox resulting in a expanding list of duplicate values when refreshed multiple times. Please adjust.
![connectionWindowComboBox](https://github.com/user-attachments/assets/bf91f186-c051-4f1f-b35f-312851f98369)

Reply:
you didnt fix this still.